modules/ossm-creating-certificates-for-multi-cluster-mesh.adoc

```diff
@@ -0,0 +1,197 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-creating-certificates-for-a-multi-cluster-mesh_{context}"]
+= Creating certificates for a multi-cluster mesh 
+
+This procedure explains how to create root and intermediate certificates for two clusters, named East and West.
+
+[NOTE]
+====
+In this procedure, `cluster1` is the East cluster and `cluster2` is the West cluster. 
+====
+
+.Prerequisites
+
+* You have OpenSSL installed locally.
+
+.Procedure
+
+. Create a root certificate file named `root-ca.conf`:
++
+.Sample root certificate file
+[source,terminal]
+----
+encrypt_key = no
+prompt = no
+utf8 = yes
+default_md = sha256
+default_bits = 4096
+req_extensions = req_ext
+x509_extensions = req_ext
+distinguished_name = req_dn
+[ req_ext ]
+subjectKeyIdentifier = hash
+basicConstraints = critical, CA:true
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, keyCertSign
+[ req_dn ]
+O = Istio
+CN = Root CA
+----
+
+. Create a key for the root certificate by running the following command:
++
+[source,terminal]
+----
+$ openssl genrsa -out root-key.pem 4096
```

Comment:
Do commands need to start with `$ `? This makes the copy function (at the top right of each code snippet) less useful, because the user then needs to remove the $ sign manually.

Reply:
Yes, all command must start with the $ prompt.

Reply:
Then you'll have to add the prompt to a bunch of other commands in this PR. A lot of them don't have the prompt.

Also, you might want to suggest to either remove the copy icon from the snippets or change it so that it automatically removes the `$` sign when copying.

Reply:
published doc does not have this problem